src/main/java/com/devcycle/sdk/server/local/managers/EventQueueManager.java

```diff
@@ -105,6 +110,36 @@ public void queueEvent(DevCycleUser user, DevCycleEvent event) throws Exception
         this.localBucketing.queueEvent(this.sdkKey, OBJECT_MAPPER.writeValueAsString(user), OBJECT_MAPPER.writeValueAsString(event));
     }
 
+    public void queueSDKConfigEvent(Request req, Response<ProjectConfig> response) throws Exception {
+        DevCycleUser user = new DevCycleUser();
+        user.setUserId(localBucketing.getClientUUID() + "@" + InetAddress.getLocalHost().getHostName());
+        DevCycleEvent event = new DevCycleEvent();
+        event.setType("sdkConfig");
+        event.setTarget(req.url().toString());
+
+        try(okhttp3.Response res = response.raw()) {
+            if (res.isSuccessful()) {
+                event.setValue(BigDecimal.valueOf(res.networkResponse().receivedResponseAtMillis() - res.networkResponse().sentRequestAtMillis()));
+            } else {
+                event.setValue(BigDecimal.valueOf(-1));
```

Comment:
you should still be able to get a response time if the request fails.

Reply:
we don't track the event if it's not successful - its not really needed

Reply:
not all cases return a valid response (ie timeout)

Reply:
we do want to track the event if its not successful. that's why there is a `errMsg` in the metadata.

Reply:
I can switch it up - but I think it'd be more confusing the way it'd have to be structured

Reply:
the main reason for this event is to really track the different errors the SDKs receive, so it's kinda a requirement here.